rabbitmq/cluster.sls

```diff
@@ -5,24 +5,14 @@ include:
 - linux.network.host
 - rabbitmq.server.service
 
-rabbitmq_dirs:
-  file.directory:
-  - names:
-    - /root/rabbitmq/flags
-  - mode: 700
-  - user: root
-  - group: root
-  - makedirs: true
-
 {% if cluster.get('role', 'slave') == 'master' %}
 
 rabbitmq_cluster_init:
   cmd.run:
   - names:
     - rabbitmqctl set_cluster_name {{ cluster.get('name', 'rabbitmq') }}
```

Comment:
It was enough to use this:
```
cmd.run:
  name: rabbitmqctl set_cluster_name {{ cluster.get('name', 'rabbitmq') }} && touch /var/lib/rabbitmq/.cluster-installed
  creates: /var/lib/rabbitmq/.cluster-installed
```
and remove useless rabitmq_cluster_init_final bellow.